fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -816,6 +817,23 @@ protected void allocateTenant() {
         // Fill any static data tables (which are also partitioned by tenant)
         populateStaticTables(gen, tenantKey);
 
+        // Prepopulate the Resource Type Tables
+        try (ITransaction tx = TransactionFactory.openTransaction(connectionPool)) {
+            try {
+                Connection c = connectionPool.getConnection();
```

Comment:
can we use one try here?

Reply:
it's the pattern that Robin follows through the code. thus the transaction factory, and then the nested tries

Reply:
not sure if we need to close the connection or put it back to the connection pool ...

Reply:
We do not need to put it back in the pool... from what I've reviewed in the code, however, you are correct that we need to do autocloseable.

Reply:
fixed